src/libs/ReportUtils.ts

```diff
@@ -10754,6 +10753,15 @@ function getReportPersonalDetailsParticipants(report: Report, personalDetailsPar
     };
 }
 
+function getReportAttributes(reportID: string | undefined, reportAttributes?: ReportAttributesDerivedValue['reports']) {
+    const attributes = reportAttributes ?? reportAttributesDerivedValue;
+
+    if (!reportID || !attributes?.[reportID]) {
+        return;
+    }
+    return attributes[reportID];
+}
+
```

Comment:
I think we dont want to create these getters in general and always use the attributes fetched from onyx in the component

Reply:
Addressed this comment here: https://github.com/Expensify/App/pull/62464